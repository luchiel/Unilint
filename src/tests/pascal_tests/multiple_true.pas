var
  i, j: integer;
begin
  i := j; j := i; {hello! I'm a comment!} i := j;
 end.
  