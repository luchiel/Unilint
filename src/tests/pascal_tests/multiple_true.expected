4:11: two or more expressions at one line
4:43: two or more expressions at one line
